src/pages/SharedPage.js

```diff
@@ -22,19 +20,18 @@ const Page = () => {
   };
 
   useEffect(() => {
-    handleLoad(getUser, setUser);
-    handleLoad(getFolder, setFolder);
+    handleLoad(getSharedUser, setUser);
+    handleLoad(getSharedFolder, setFolder);
   }, []);
 
   return (
     <div>
-      <Header userInfo={user} />
+      <Header userInfo={user} isSticky={true} />
```

Comment:
P3:
제 개인적인 취향은 아래와 같습니다..

```suggestion
      <Header userInfo={user} isSticky />
```